types/features.ts

```diff
@@ -5,10 +5,20 @@ export interface RSSFeed {
   blogCategory?: string;
 }
 
-export interface WebsiteBanner {
+export interface WithRange {
```

Comment:
Does this need to be exported still?

Reply:
The WithRange? hmmm not really

Reply:
Yeah, I figured it probably wouldn't be consumed outside inheritance within this file?

Reply:
Updated :)